src/pages/blog/cdn.md

```diff
@@ -4,68 +4,68 @@ image: 'https://i.imgur.com/VsS5RwW.png'
+When you query against [Microlink API](/docs/api/getting-started/overview), the first time you query for a resource that wasn't previously served it will be created, what is known as a cache *MISS*.
 
-The successive requests for the resource will consume the cached version of the resource, what is known as cache *HIT*.
+The successive requests for the resource will consume the cached version, what is known as a cache *HIT*.
 
 ## What's new
 
-We revisited the cache layer and implemented some improvements, doing it more powerful than ever.
+We revisited the cache layer and implemented some improvements, making it more powerful than ever.
 
-## Cache saves your API quota plan
+## Cache saves your API quota
 
 <Terminal>
   npx microlink-api https://example.com&screenshot&embed=screenshot.url
 </Terminal>
 
-**Starting from today, any response served from the cache will not count as part of your API quota plan.**
+**Starting from today, any response served from the cache won't count towards your API quota.**
 
-This is a drastically change how users consume your API. Let me clarify with an example.
+This drastically changes how users consume the API. Let me clarify with an example.
 
-One of the [Microlink API](/docs/api/getting-started/overview) use cases is [embed](/docs/api/parameters/embed) it directly in your HTML markup:
+One of the [Microlink API](/docs/api/getting-started/overview) use cases is to [embed](/docs/api/parameters/embed) it directly in your HTML markup:
 
 ```html
 <meta name="og:image" content="https://api.microlink.io?url=https://example.com&screenshot&embed=screenshot.url">
 ```
 
 <Figcaption>A screenshot generated on the fly, always up to date.</Figcaption>
 
-In the past, if you do this and your website has *1000 pageviews*, means you consumed *1000 requests* from your API quota.
+In the past, if you did this and your website had *1000 pageviews*, it meant you consumed *1000 requests* from your API quota.
 
-Now, you only consume \*one request\*, serving the rest from cache and **not counting them in your API quota plan**.
+Now, you only consume \*one request\*, serving the rest from the cache and **not counting them in your API quota plan**.
 
 ## Cache is served around the world
 
 ![](https://blog-cloudflare-com-assets.storage.googleapis.com/2019/08/image1-3.png)
 
-<Figcaption>CloudFlare network has more than 240 edge servers over 90 countries.</Figcaption>
+<Figcaption>The CloudFlare Network has more than 240 edge servers in over 90 countries.</Figcaption>
 
-The first time you hit [Microlink API](/docs/api/getting-started/overview), the request will reply by a server located somewhere.
+The first time you hit [Microlink API](/docs/api/getting-started/overview), the response will come from one of the servers located somewhere.
```

Comment:
@Kikobeats I don't really like the "located somewhere" bit, but I'm not too sure what you want to say exactly.

Write it in Spanish if you want and I'll suggest how _I'd_ express it 👌